### ISSUE TITLE
Add DeepAlpha - AI crypto trading bot

### ISSUE THREAD

**Issue by stefanoviana** [igorbarinov/awesome-blockchain#20]
Hi! I would like to suggest adding [DeepAlpha](https://github.com/stefanoviana/deepalpha) to this list.

**What is DeepAlpha?**
AI crypto trading bot with 70.9% walk-forward validated accuracy. XGBoost + LightGBM ensemble with 72 features. MIT license.

**Suggested entry:**
```
- [DeepAlpha](https://github.com/stefanoviana/deepalpha) - AI crypto trading bot with 70.9% walk-forward validated accuracy. XGBoost + LightGBM ensemble with 72 features. MIT license.
```

Thank you!
